mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -177,31 +358,79 @@ pub(super) async fn init_light_client(
             Ok(db)
         }
     })
-    .await?;
+    .await
+}
+
+pub(super) async fn init_light_client(
+    _lightwalletd_urls: Vec<String>,
+    lightwalletd_url: Uri,
```

Comment:
Having two arguments seems confusing. You can use `Vec<String>` only and try to parse `Uri` from the first element of the vector.

Reply:
done